src/commands/diff.ts

```diff
@@ -26,14 +26,20 @@ export default class Diff extends Command {
       char: 'f',
       description: 'format of the output',
       default: 'yaml',
-      options: ['json', 'yaml', 'yml'],
+      options: ['json', 'yaml', 'yml', 'markdown', 'md'],
```

Comment:
```suggestion
      options: ['json', 'yaml', 'yml', 'md'],
```

We should have just the `md` option.